app.js

```diff
@@ -26,10 +26,10 @@ const server = http.createServer(app);
 
 const redis = require('redis');
 const redisClient = redis.createClient({
-	port: process.env.REDIS_PORT || undefined,
-	host: process.env.REDIS_HOST || undefined,
-	password: process.env.REDIS_PASSWORD || undefined,
-	db: process.env.REDIS_DB || undefined,
+	port: process.env.REDIS_PORT || "6379",
+	host: process.env.REDIS_HOST || "127.0.0.1",
+	password: process.env.REDIS_PASSWORD || "",
+	db: process.env.REDIS_DB || "redis",
```

Comment:
Don't change these defaults. They let the dev environment connect to a redis instance on the same machine automagically

Reply:
I had an issue I try right now to fix it 
```
opentogethertube_dev    | [api-server] error event - undefined:undefined - Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379
opentogethertube_dev    | [api-server] Generated name for new user (on request): skinny_teal_hippopotamus
opentogethertube_dev    | [api-server] > GET / {}
```